extensions/scarb-doc/src/docs_generation/markdown/book_toml.rs

```diff
@@ -0,0 +1,16 @@
+use indoc::formatdoc;
+
+use crate::types::Crate;
+
+pub(super) fn generate_book_toml_content(crate_: &Crate) -> String {
+    formatdoc! {"
+        [book]
+        authors = [\"<unknown>\"]
+        language = \"en\"
+        multilingual = false
+        src = \"src\"
+        title = \"Documentation of {}\"
```

Comment:
I think it's better to put `{}` first so that it'll be obvious which crate's documentation this is in browser tabs:

<img width="743" alt="image" src="https://github.com/user-attachments/assets/529dafe8-2d03-4598-869d-583f5a580ac2">

Reply:
Done